src/utils/fetch-graphql.js

```diff
@@ -206,7 +206,7 @@ const handleFetchErrors = async ({
       formatLogMessage(
         `It took too long for ${url} to respond (longer than ${
           timeout / 1000
-        } seconds). Either your URL is wrong, you need to increase server resources, or you need to increase your timeout in the gatsby-source-wordpress options. \n${genericError(
+        } seconds). Either your URL is wrong, you need to increase server resources, you need to decrease your options.schema.perPage value from the default 100, or you need to increase your timeout in the gatsby-source-wordpress options. \n${genericError(
```

Comment:
Hey Jacob, thanks for this! I have a feeling it gets hard to read when the list of things grows beyond 3. What do you think about something like this?
```suggestion
        } seconds). Either your URL is wrong, you need to increase server resources, or you need to decrease the amount of resources each request takes. You can configure how much resources each request takes by lowering your `options.schema.perPage` value from the default of 100 nodes per request. Alternatively you can increase the request timeout by setting a value in milliseconds to `options.schema.timeout`, the current setting is ${timeout}. \n${genericError(
```

Reply:
Oh yeah, that works!